src/app/components/home/home.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, OnInit } from '@angular/core';
+import { GlobalDataService } from 'src/app/globalData.service';
+import { IGlobal } from 'src/app/global';
+import { ICountries } from 'src/app/countries';
+import { GoogleChartInterface } from 'ng2-google-charts';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.scss'],
+})
+export class HomeComponent implements OnInit {
+  Date = '';
+  Country = '';
+  dataCountries: ICountries[];
+  dataGlobal: IGlobal;
+  dataServiceSub$: Subscription;
+  isTableShow = true;
+
+  pieChart: GoogleChartInterface = {
+    chartType: 'PieChart',
+  };
+  columnChart: GoogleChartInterface = {
+    chartType: 'ColumnChart',
+  };
+
+  constructor(private dataService: GlobalDataService) {}
+
+  ngOnInit() {
+    this.fetchData();
+  }
+
+  fetchData() {
+    this.dataServiceSub$ = this.dataService
+      .fetchGlobalData()
+      .subscribe(({ countries, global }) => {
+        this.dataCountries = countries;
+        this.dataGlobal = global;
+
+        countries.map((el) => {
+          this.Date = el.Date;
+        });
+        this.initChart();
+      });
+  }
+
+  ngOnDestroy() {
```

Comment:
^ Move this under the ngOnInit()

Reply:
Component order: 

- @Input @Output
- Properties
- Constructor
- Lifecycle methods 
- Custom methods